scripts/docker-run.sh

```
@@ -0,0 +1,104 @@
+# Check for help
+if [[ "$1" == "--help" ]] || [[ "$1" == "-h" ]] || [[ -z "$1" ]]; then
+  show_help
+fi
+
+# Get network
+NETWORK="$1"
+shift
+
+# Get command if provided
+if [ $# -gt 0 ]; then
+  COMMAND="$@"
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)